src/d3d_prefida.pro

```diff
@@ -7,11 +7,13 @@ PRO d3d_prefida, inputs, igrid=igrid,bgrid=bgrid
                    ab:2.01410178d0,ai:2.0141078d0,impurity_charge:6,$
                    lambdamin:647.0d0,lambdamax:667.0d0,nlambda:2000,$
                    n_fida:5000000L,n_npa:500000L,n_nbi:50000L, $
+                   n_pfida:50000000L,n_pnpa:50000000L, $
                    n_halo:500000L,n_dcx:500000L,n_birth:10000L,$
                    ne_wght:50,np_wght:50,nphi_wght:100,emax_wght:100.0d0,$
                    nlambda_wght:1000,lambdamin_wght:647.d0,lambdamax_wght:667.d0,$
-                   calc_npa:0,calc_brems:0,calc_bes:0,calc_fida:0,$
-                   calc_birth:0,calc_fida_wght:0,calc_npa_wght:0,dump_dcx:0,$
+                   calc_npa:2,calc_brems:1,calc_fida:1,calc_neutron:1,$
+                   calc_nbi:1,calc_dcx:1,calc_halo:1,calc_cold:1,$
+                   calc_birth:1,calc_fida_wght:1,calc_npa_wght:1,calc_pfida:1,calc_pnpa:2,$
```

Comment:
By default everything should be turned off. Only turn things on when you are sure the user provided the right diagnostics. For instance its possible to run FIDASIM without any diagnostics supplied.